fairlearn/adversarial/_adversarial_mitigation.py

```diff
@@ -341,39 +342,7 @@ def __setup(self, X, y, A):
 
         def read_kw(data, kw_or_func, kwname):
```

Comment:
Indeed, I agree don't need it. I started to remove it before posting the PR, but decided last minute to leave it for a future refactor. I think it can fit this PR nicely though, the rest can follow later. I will push a commit and ping again.

Reply:
addressed in [d52a15c](https://github.com/fairlearn/fairlearn/pull/1399/commits/d52a15ca2597f3504fc37f34bfcae370b5efa92d)